pkg/playground/test.html

```diff
@@ -45,6 +45,18 @@
 			<p>cockpit.user() information</p>
 			<div id="user-info" />
                 </div>
+                <br/>
+                <div id="fsreplace1-div">
+                    <h2>fsreplace1 test</h2>
+                    <p>new filename</p>
+                    <input id="fsreplace1-filename" />
+                    <p>text content</p>
+                    <input id="fsreplace1-content" />
+                    <input id="fsreplace1-append" type="checkbox" />
+		    <label for="fsreplace1-append">Use existing tag</label>
```

Comment:
Calling this "append" is rather confusing -- this has nothing to do with appending the file. "-tag"?